app/helpers/session_timeout_warning_helper.rb

```diff
@@ -11,6 +11,13 @@ def warning
     (Figaro.env.session_timeout_warning_seconds || 30).to_i
   end
 
+  def timeout_refresh_url
+    URI(request.original_url).tap do |url|
+      query = Rack::Utils.parse_nested_query(url.query).with_indifferent_access
+      url.query = query.merge(timeout: true).to_query
+    end.to_s.html_safe # rubocop:disable Rails/OutputSafety
+  end
```

Comment:
Do we have a good way to systematically figure out which pages/routes have forms and which do not?